exonum-java-binding/core/rust/ejb-app/src/node_builder.rs

```diff
@@ -191,4 +188,20 @@ mod tests {
         writeln!(cfg_file, "{}", cfg).unwrap();
         cfg_file.into_temp_path()
     }
+
+    // Checks whether particular service is presented in the given collection.
+    fn contains_service(service_name: &str, factories: &[Box<dyn ServiceFactory>]) -> bool {
+        let name = if service_name == CONFIGURATION_SERVICE
```

Comment:
I find this too fancy for a test utility method than a bunch of constants. Can't we use just the constants directly in assertions without these conversions (assert(contains_service(CONFIGURATION_FACTORY_NAME, ...)?

Reply:
Well, the previous variant with HashMap was used with constants directly with assertions, but it was a little overcomplicated for production code. With just a vector we have to iterate over it anyway, so this is the reason for `contains_service` method. Would it be better to make it `assert(contains_service(CONSTANT, collection))` -> `assert_contains_service(CONSTANT, collection)`?

Reply:
No, I'd consider using the constants defined in each service in assertions instead of translating them with this conditional.